gym/core.py

```diff
@@ -142,64 +156,66 @@ def render(self, mode="human"):
               the list of supported modes. It's recommended to call super()
               in implementations to use the functionality of this method.
 
-        Args:
-            mode (str): the mode to render with
-
-        Example::
-
-            class MyEnv(Env):
-                metadata = {'render_modes': ['human', 'rgb_array']}
+        Example:
+            >>> class MyEnv(Env):
+            ...    metadata = {'render_modes': ['human', 'rgb_array']}
+            ...
+            ...    def render(self, mode='human'):
+            ...        if mode == 'rgb_array':
+            ...            return np.array(...) # return RGB frame suitable for video
+            ...        elif mode == 'human':
+            ...            ... # pop up a window and render
+            ...        else:
+            ...            super(MyEnv, self).render(mode=mode) # just raise an exception
 
-                def render(self, mode='human'):
-                    if mode == 'rgb_array':
-                        return np.array(...) # return RGB frame suitable for video
-                    elif mode == 'human':
-                        ... # pop up a window and render
-                    else:
-                        super(MyEnv, self).render(mode=mode) # just raise an exception
+        Args:
+            mode: the mode to render with, valid modes are `env.metadata["render_modes"]`
         """
         raise NotImplementedError
 
     def close(self):
         """Override close in your subclass to perform any necessary cleanup.
 
-        Environments will automatically close() themselves when
+        Environments will automatically :meth:`close()` themselves when
         garbage collected or when the program exits.
         """
         pass
 
     def seed(self, seed=None):
-        """Sets the seed for this env's random number generator(s).
+        """Deprecated function that sets the seed for the environment's random number generator(s).
+
+        Use :meth:`env.reset(seed=seed)` as the new API for setting the seed of the environment.
```

Comment:
:meth:`env.reset(seed=seed)` does not show up correctly on the website
![image](https://user-images.githubusercontent.com/15806078/167289933-f57dac29-5c4a-4c9f-9b0f-9c4e45ec2be9.png)